src/Schema/EnumSchema.php

```diff
@@ -29,6 +31,31 @@ public function toArray(): array
         return [
             'description' => $this->description,
             'enum' => $this->options,
+            'type' => $this->getOptionTypes(),
         ];
     }
+
+    /**
+     * @return string|array<int, string>
+     */
+    protected function getOptionTypes(): array|string
+    {
+        // @phpstan-ignore return.type
```

Comment:
Without this, I'm getting this error:

```
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   Schema/EnumSchema.php                                                                                                                                                                
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  43     Method EchoLabs\Prism\Schema\EnumSchema::getOptionTypes() should return array<int, string>|string but returns array<int, string>|Illuminate\Support\Collection<int, string>|string.  
         🪪  return.type                                                                                                                                                                      
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
```

I would personally probably forego the last Collection::when() call, since I don't think it makes it easier to read, and is throwing static analysis for a loop.

```php
$types = collect($this->options)
    ->map(fn (mixed $option): string => match (gettype($option)) {
        'integer', 'double' => 'number',
        'string' => 'string'
    })
    ->when(
        $this->nullable,
        fn (Collection $collection): Collection => $collection->push('null')
    )
    ->unique()
    ->values();

if ($types->count() === 1) {
    return $types[0];
}

return $types->all();
```

Let me know your thoughts @sixlive.